server/start.go

```diff
@@ -89,14 +89,17 @@ func startStandAlone(ctx *Context, appCreator AppCreator) error {
 		cmn.Exit(err.Error())
 	}
 
-	// wait forever
 	cmn.TrapSignal(ctx.Logger, func() {
 		// cleanup
 		err = svr.Stop()
 		if err != nil {
 			cmn.Exit(err.Error())
 		}
 	})
+
+	// run forever (the node will not be returned)
+	select {}
```

Comment:
I'm not sure what standalone is used for, but having a while loop spinning is going to burn a lot of cpu cycles here for no reason. Is the point to just wait for a signal? Why not just call Wait on the server?